code/modules/antagonists/changeling/powers/headcrab.dm

```diff
@@ -32,6 +32,8 @@
 		S.Paralyze(60)
 	var/turf = get_turf(user)
 	user.gib()
+	if (!user.stat && prob(33))
+		new /obj/effect/decal/cleanable/blood/gibs/changeling(turf)
```

Comment:
```suggestion
	if (prob(33))
		new /obj/effect/decal/cleanable/blood/gibs/changeling(turf)
```

Why do you check `user`'s `stat` when the `user` will be clearly dead after `gib` is called?

Reply:
Huh. The way this is meant to function is it will check if the user is already dead at the moment. If the user is dead/regen coma, there will be a high chance to make changeling gibs.

Reply:
This is headcrab function, i.e. "Last Resort", where changeling *should* die and be reborn as a headcrab. Changeling dies as `user.gib()` is called and gets reborn as headcrab at `var/mob/living/simple_animal/hostile/headcrab/crab = new(turf)` and following lines.

Reply:
You're right. I moved that function above user.gib(). Thanks for pointing it out.